shared/charts/gpii-flowmanager/templates/certificate.yaml

```diff
@@ -6,13 +6,13 @@ metadata:
 spec:
   secretName: {{ template "flowmanager.name" . }}-tls
   issuerRef:
-    name: {{ .Values.issuerRef.name }}
-    kind: {{ .Values.issuerRef.kind }}
+    name: letsencrypt-{{ template "flowmanager.name" . }}
+    kind: Issuer
   dnsNames:
   {{ toYaml .Values.dnsNames | indent 2 }}
   acme:
     config:
-    - http01:
-        ingressClass: nginx
+    - dns01:
```

Comment:
Should we keep HTTP challenge as option in charts?

Reply:
Why? We're not planning to use it in any environment - it actually does not work with Istio (as it requires a) Ingress and b) the DNS pointed to that Ingress). I prefer to remove the https, instead vs. complicating chart with extra logic that is never used.